torch/_dynamo/eval_frame.py

```diff
@@ -148,6 +149,13 @@ def __dir__(self):
             attr for attr in super().__dir__() if attr not in orig_mod_attrs
         ]
 
+    def get_compiler_config(self):
```

Comment:
Inductor-specific code shouldn't go in `/_dynamo/`, we should try to keep things backend agnostic.

Reply:
Got you. I updated the diff with ```backend.get_compiler_config()``` so it becomes more backend agnostic